packages/components/snackbar/src/Snackbar.stories.tsx

```diff
@@ -217,17 +217,32 @@ export const Action: StoryObj = {
       <div>
         <Snackbar />
 
-        <Button
-          onClick={() =>
-            addSnackbar({
-              message: "You're done!",
-              actionLabel: 'Undo',
-              onAction: () => console.log('Undone'),
-            })
-          }
-        >
-          Display snackbar
-        </Button>
+        <div className="grid grid-cols-1 gap-xl md:grid-cols-2">
+          <Button
+            onClick={() =>
+              addSnackbar({
+                message: "You're done!",
+                actionLabel: 'Undo',
+                onAction: () => console.log('Undone'),
+              })
+            }
+          >
+            Display snackbar with action (default)
+          </Button>
+
+          <Button
+            onClick={() =>
+              addSnackbar({
+                message: "You're done! But maybe you should care about what you just did?",
+                actionLabel: 'Undo',
+                onAction: () => console.log('Undone'),
+                forceActionOnNewline: true,
```

Comment:
I would just remove `force` -> `actionOnNewLine`

Reply:
No strong opinion about this :neutral_face: Wdyt @andresin87 ?

Reply:
Not a big issue if we keep as it is, it's just that "force" make it sound like an "undesired" feature.

Reply:
Ok then :+1: